### PR TITLE
fix: schema eth, country code validation

### PR DESCRIPTION
**Pull request by Alexander-frenki**

### CODE REVIEW

File: ui/src/components/credentials/IssueCredentialForm.tsx [hunk 1]
```diff
@@ -144,6 +144,11 @@ export function IssueCredentialForm({
     }
   };
 
+  const isCountryCode = (x: number) => {
+    const iso31661NumericRegex = /^\d{1,3}$/;
+    return iso31661NumericRegex.test(x.toString());
```
Comment (amonsosanz) [inline ui/src/components/credentials/IssueCredentialForm.tsx:149]: This will validate any 3-digit number, not valid country codes.

Comment (Alexander-frenki) [reply ui/src/components/credentials/IssueCredentialForm.tsx:149]: I know, but we decided that this is  enough in this case, and using a list of countries will be redundant now

Comment (demonsh) [reply ui/src/components/credentials/IssueCredentialForm.tsx:149]: Hi @amonsosanz, if you have more advanced validator please share it. We will added it to the node. 
But at the moment to allow testing, we will add just minor validation of 3 digits.

Comment (demonsh) [reply ui/src/components/credentials/IssueCredentialForm.tsx:149]: We will add it as separate pull reqeust

Comment (amonsosanz) [reply ui/src/components/credentials/IssueCredentialForm.tsx:149]: This is how we validate it in the schema builder:

https://github.com/0xPolygonID/schema-builder/pull/243/files#diff-2c90d10e1dc2218567877731655acf2af77eec2b038fbf5233a6806aee7d20b9

```
    const isCountryCode = (code: number) => {
      return iso31661Countries.find((country) => country.code === code) !== undefined;
    };
```

**Pull request merged**